docs/examples/fp8_primer.ipynb

```diff
@@ -100,19 +100,66 @@
     "</figure>"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "fd7b4f37-50a2-4d41-9067-cf0c471cb2d7",
+   "metadata": {},
+   "source": [
+    "## Beyond FP8 - training with NVFP4\n",
+    "\n",
+    "In addition to MXFP8, NVIDIA Blackwell introduced support for an even smaller, 4-bit format called NVFP4. The values are represented there in E2M1 format, able to represent values of magnitude up to +/-6.\n",
+    "\n",
+    "<figure align=\"center\" id=\"fig_8\">\n",
+    "<img src=\"FP4_format.png\" width=\"50%\">\n",
+    "<figcaption> Figure 8: FP4 E2M1 format can represent values between +/-6.</figcaption>\n",
+    "</figure>\n",
+    "\n",
+    "### NVFP4 Format\n",
+    "\n",
+    "NVFP4 format is similar to MXFP8 - it also uses granular scaling to preserve the dynamic range. The differences are:\n",
+    "\n",
+    " - the granularity of the scaling factors: in NVFP4 format a single scaling factor is used per block of 16 elements, whereas MXFP8 uses 1 scaling factor per block of 32 elements\n",
+    " - the datatype of the scaling factors: NVFP4 uses FP8 E4M3 as the scaling factor per block, whereas MXFP8 uses E8M0 as the scaling factor datatype. Choice of E4M3 for the scaling factor enables preservation of more information about mantissa, but does not enable the full dynamic range of FP32. Therefore, NVFP4 uses an additional single per-tensor FP32 scaling factor to avoid overflows.\n",
```

Comment:
```suggestion
    " - Granularity of the scaling factors: in NVFP4 format a single scaling factor is used per block of 16 elements, whereas MXFP8 uses 1 scaling factor per block of 32 elements\n",
    " - Datatype of the scaling factors: NVFP4 uses FP8 E4M3 as the scaling factor per block, whereas MXFP8 uses E8M0 as the scaling factor datatype. Choice of E4M3 for the scaling factor enables preservation of more information about mantissa, but does not enable the full dynamic range of FP32. Therefore, NVFP4 uses an additional single per-tensor FP32 scaling factor to avoid overflows.\n",
```